packages/x-components/src/x-modules/related-prompts/wiring.ts

```diff
@@ -116,6 +124,7 @@ export const relatedPromptsWiring = createWiring({
     setSelectedQueryWire
   },
   UserAcceptedAQueryPreview: {
+    setRelatedPromptsQueryFromQueryPreview,
```

Comment:
```suggestion
    setRelatedPromptsQueryFromUrl,
```

Why don't you use the existing `setRelatedPromptsQueryFromUrl`?

Reply:
Because the naming doesn't match with the purpose. But maybe I can rename `setRelatedPromptsQueryFromUrl`

Reply:
Yes, I would go with it 🚀